jubeatsaucer.html

```diff
@@ -0,0 +1,33 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>jubeat saucer DLL Modder</title>
+        <link rel="stylesheet" href="css/style.css">
+        <!-- don't hate -->
+        <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
+        <script type="text/javascript" src="js/FileSaver.min.js"></script>
+        <script type="text/javascript" src="js/dllpatcher.js"></script>
+        <script type="text/javascript">
+            window.addEventListener("load", function() {
+                new PatchContainer([
+                    new Patcher("jubeat.dll", "jubeat.dll, 2014-01-28", [
```

Comment:
Don't put "jubeat.dll" in the same string that has the date - just `"jubeat.dll", "2014-01-28"`

Same for the mdb patch, and saucer fulfill.

Patches are otherwise ok 👍